### ISSUE TITLE
Dropdown is not disabled with the HTML5 disabled attribute

### ISSUE THREAD

**Issue by scottwernervt** [cibernox/ember-basic-dropdown#298]
Pull request #44 introduced `aria-disabled` attribute when `{{#basic-dropdown disabled=true as |dd|}}`. HTML5 has a `disabled` attribute for [disabled elements](https://www.w3.org/TR/html5/disabled-elements.html). It should be added to basic-dropdown component / template.

**Comment by cibernox** [comment]
@scottwernervt I **believe** that `disabled` can't be applied to the trigger of this dropdown, as its not an input, but I am no expect on this topic. If it has `role=button` then it's "legal"?

I'd be glad to add it if I'm wrong.

**Comment by scottwernervt** [comment]
You are correct, `disabled` would only apply if you override the trigger's `tagName` to a `button`.

```hbs
{{#basic-dropdown disabled=true as |dd|}}
  {{#dd.trigger tagName="button" class="trigger-bootstrap-feel"}}Click me{{/dd.trigger}}

  {{#dd.content class="content-bootstrap-feel"}}
    You can't see this
  {{/dd.content}}
{{/basic-dropdown}}
```
Framework trigger tags for dropdowns:
* [semantic-ui](https://semantic-ui.com/modules/dropdown.html) `div`
* [bootstrap v4](http://v4-alpha.getbootstrap.com/components/dropdowns/) `button` or `a`
* [zurb foundation](http://foundation.zurb.com/sites/docs/dropdown.html) `button`
* [spectre](https://picturepan2.github.io/spectre/components.html#menus) `a`
* [bulma](https://github.com/jgthms/bulma/pull/546/files/cf0839d59d6ff480c319609fbea302f5d7e2e88e#diff-7f3b97bce5bd5d31c1e9c0c646689372) `button`

I will just extend my CSS to support `button[aria-disabled="true"]`  since `disabled` would only be valid for a `button` and it is not worth the extra code logic for this single case.

**Comment by artemgurzhii** [comment]
@cibernox I'm having kind of the same problem, here is my `hbs`

```hbs
{{#basic-dropdown as |dropdown|}}
  {{#dropdown.trigger
    tagName="button"
    type="button"
    role="button"
    disabled=someBooleanProp
    aria-disabled=someBooleanProp
  }}
    <i class="some-icon"></i>
  {{/dropdown.trigger}}

  {{#dropdown.content}}
    content goes here
  {{/dropdown.content}}
{{/cm-popover}}
```

Everything is working as expected except the `disabled` attribute. Here is the result:
```html
<button aria-disabled role="button" type="button"></button>
```

Is it intended to work like that?
If yes, is there a way to still propagate `disabled` property to the dropdown trigger?
